zerver/openapi/zulip.yaml

```
@@ -12140,13 +12142,14 @@ paths:
                           See [PATCH /settings](/api/update-settings) for details on
                           the meaning of this setting.
 
-                          **Changes**: Deprecated `realm_name_in_notifications` in Zulip 5.0 (feature level 89).
-                          Replaced `realm_name_in_notifications` boolean with
-                          `realm_name_in_email_notifications_policy` in Zulip 7.0 (feature level 168);
-                          `true` corresponded to `Always`, and `false` to `Never`.
+                          **Changes**: In Zulip 7.0 (feature level 168), replaced previous
+                          `realm_name_in_notifications` global notifications setting with
+                          `realm_name_in_email_notifications_policy`.
 
-                          Clients connecting to newer servers should declare the `user_settings_object`
-                          client capability and access the `user_settings` object instead.
+                          **Deprecated** previous `realm_name_in_notifications` in Zulip 5.0
+                          (feature level 89). Clients connecting to newer servers should declare the
```

This property is a confusing one because we deprecated it, and then later renamed it while keeping it equally deprecated.

I think the main thing I'd want here is to avoid creating ambiguity that suggests this one might be less deprecated than the other user settings documented next to it. The existing text isn't totally clear about this, but I think it helps itself on that front by cheating on our practice of putting the newest change first, in order to put the word "Deprecated" at the start of the changes. I like the use of bold on "Deprecated" here to make it stand out despite not being at the start.

Perhaps just make the continuing deprecation more explicit, something like:
```suggestion
                          **Deprecated** since Zulip 5.0 (feature level 89);
                          both `realm_name_in_notifications` and the newer
                          `realm_name_in_email_notifications_policy` are deprecated.
                          Clients connecting to newer servers should declare the
```

---

Yeah, hopefully this will be a rare case, but feels good to set a clear precedent about what to do here.

Adding that extra emphasis on the deprecated status makes sense to me. Updated as such, thanks!